src/pages/Registration/index.tsx

```diff
@@ -156,15 +154,14 @@ export default function Registration() : JSX.Element {
             ? profilePicture = `${uuidv4()}.${profilePictureFileExtension}`
             : profilePicture = uuidv4();
 
-          await storage()
-            .ref(Values.IMAGE_DIRECTORY)
+          await userAPI.userStorageProfilePictureDir()
```

Comment:
Could be `Users.putProfilePicture(profilePicturePath)` that uploads and returns uploaded filename

Reply:
Fixed by new commit.